resource-managers/kubernetes/docker/src/main/dockerfiles/spark/bindings/python/Dockerfile

```diff
@@ -32,6 +32,7 @@ RUN mkdir ${SPARK_HOME}/python
 RUN apt-get update && \
     apt install -y python3 python3-pip && \
     pip3 install --upgrade pip setuptools && \
+    pip3 install 'numpy>=1.20.0' pyarrow pandas scipy unittest-xml-reporting 'lxml==4.9.4' 'grpcio==1.67.0' 'grpcio-status==1.67.0' 'protobuf==5.28.3' && \
```

Comment:
@dongjoon-hyun I think I need this dependencies for testing ... should I create a new Dockerfile? or is it fine to just put it here?

Reply:
Seems like this is used as a built-in Dockerfile. Let me exclude it for now as we verified that tests pass anyway